packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/plan-upgrade/component.js

```diff
@@ -0,0 +1,12 @@
+import controller from './controller';
+import template from './template.html';
+
+export default {
+  bindings: {
+    onChange: '&',
```

Comment:
May be we can have a 2-way binding (like `model: '<'`) to make things simpler :)